internal/cmd/issue/edit/edit.go

```diff
@@ -341,7 +343,7 @@ func setFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("body", "b", "", "Edit description")
 	cmd.Flags().StringP("priority", "y", "", "Edit priority")
 	cmd.Flags().StringP("assignee", "a", "", "Edit assignee (email or display name)")
-	cmd.Flags().StringArrayP("label", "l", []string{}, "Replace labels")
+	cmd.Flags().StringArrayP("label", "l", []string{}, "Add labels to existed list")
```

Comment:
Sorry, forgot to submit.

```suggestion
	cmd.Flags().StringArrayP("label", "l", []string{}, "Append labels")
```